widgetssdk/src/main/res/values/legacy_strings.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is `legacy_strings.xml` file used anywhere? I cannot find any usages for some reason.

Reply:
Legacy strings are being referenced by the new strings to avoid problems with overridden strings, but I think we need to rename the files to achieve that safety. Thank you for making me think about this 😄